source/time_tables/tests/CMakeLists.txt

```diff
@@ -5,21 +5,24 @@ SET(BOOST_LIBS ${Boost_UNIT_TEST_FRAMEWORK_LIBRARY} ${Boost_FILESYSTEM_LIBRARY}
 
 
 add_executable(thermometer_test thermometer.cpp)
-target_link_libraries(thermometer_test time_tables ptreferential ed data fare routing utils pb_lib thermometer georef ${BOOST_LIBS} log4cplus protobuf)
+target_link_libraries(thermometer_test thermometer ${BOOST_LIBS})
 ADD_BOOST_TEST(thermometer_test)
 
-
+set(TIME_TABLE_LINK_LIBS
+    time_tables
+    ed
+    data
+    ${BOOST_LIBS}
```

Comment:
Seems like too many libs are into this, filesystem, regex and iostream I'm not sure